app/pb_kits/playbook/pb_checkbox/_checkbox.scss

```diff
@@ -0,0 +1,66 @@
+@import "../tokens/colors";
+$transition: 0.2s cubic-bezier(0.95, 0.05, 0.795, 0.035);
+
+[class^=pb_checkbox_kit] {
+  .pb_checkbox_label {
+    display: block;
+    position: relative;
+    padding-left: 30px;
+    cursor: pointer;
+    font-size: $text_lt_default;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
```

Comment:
I'm not sure we need these browser prefixes https://caniuse.com/#feat=user-select-none